docs/synthetic-tokens/long-short-pair.md

```diff
@@ -53,32 +50,69 @@ The method are:
 
 ### `create`
 
-A minter can call `create` to deposit collateral to mint an equal number of long and short tokens. The `collateralPerPair` contract parameter determines the amount of collateral needed to mint each pair of long and short tokens.
+After an LSP contract has been deployed, tokens can be minted by calling the `create` function before the contract’s `expirationTimestamp`. The `create` function deposits collateral into the contract in exchange for an equal amount of long and short tokens based on the `collateralPerPair` parameter. The `collateralPerPair` parameter determines the amount of collateral that is required for each pair of long and short tokens. 
+
+![](/docs/lsp-tokens/lsp_create.png)
+
+To create tokens, first approve the token minting contract to transfer the collateral currency on your behalf. Then input the number of tokens you would like to create into the Write Contract tab of the LSP contract in Etherscan. If the collateral is WETH and the `collateralPerPair` is set to 1, each long and short token minted would require 1 WETH as collateral.
```

Comment:
Recommend adding one more example that is less intuitive - if collateralPerPair is 0.25 etc.

Reply:
Updated to include an example with .25 WETH.